R/FilterCMIM.R

```diff
@@ -19,10 +19,22 @@
 #' @template seealso_filter
 #' @export
 #' @examples
-#' task = mlr3::tsk("iris")
-#' filter = flt("cmim")
-#' filter$calculate(task, nfeat = 2)
-#' as.data.table(filter)
+#' if (requireNamespace("praznik")) {
+#'   task = mlr3::tsk("iris")
+#'   filter = flt("cmim")
+#'   filter$calculate(task, nfeat = 2)
+#'   as.data.table(filter)
+#' }
+#'
+#' if (requireNamespace("mlr3pipelines") && requireNamespace("praznik")) {
```

Comment:
I'd favor using `mlr3misc::require_namespaces(c("mlr3pipelines", "praznik"))` in cases with n > 1.

Reply:
Fine for me.

Reply:
but it needs argument `quietly = TRUE` then 😞